arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,916 @@
+        "description": "Migrates an existing Power BI Workspace Collection to a different resource group and/or subscription.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "Azure resource group",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Workspace migration request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MigrateWorkspaceCollectionRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Migration completed successfully"
+          },
+          "default": {
+            "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Error": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorDetail"
+          }
+        }
+      }
+    },
+    "ErrorDetail": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        }
+      }
+    },
+    "WorkspaceCollectionList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WorkspaceCollection"
+          }
+        }
+      }
+    },
+    "WorkspaceList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Workspace"
+          }
+        }
+      }
+    },
+    "OperationList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        }
+      }
+    },
+    "AzureSku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "SKU name",
+          "type": "string",
+          "enum": [
+            "S1"
+          ]
```

Comment:
Yes, both `name` and `tier` are required properties.  They are currently strings and might change in the future so I used this the `"modelAsString": true` to avoid validation.

```
"x-ms-enum": {
  "name": "AzureSkuName",
  "modelAsString": true
}
```